README.md

```diff
@@ -1,26 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# mobile skeleton
 
 > Description the project.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+- Major languages -css and html
```

Comment:
- [ ] As this is the time you are starting your portfolio project, let us make the project documentation more descriptive and professional. Currently, your README.md looks scanty and it is not professional and descriptive. Kindly add some sections such as technologies used to build it, How you are supposed to install the project and use it locally etc.